mem0/memory/main.py

```diff
@@ -154,7 +154,8 @@ def _add_to_vector_store(self, messages, metadata, filters):
                 logging.info(resp)
                 try:
                     if resp["event"] == "ADD":
-                        memory_id = self._create_memory(data=resp["text"], metadata=metadata)
```

Comment:
why would you remove this line?

Reply:
<img width="739" alt="Screenshot 2024-09-06 at 1 04 06 PM" src="https://github.com/user-attachments/assets/37684a4f-5cb9-4548-9888-a62634c04778">


it was giving me linting issue

Reply:
this line is required, the error is about the unused "memory_id" variable. You should keep this line otherwise no new memories will be created

Reply:
added that line again.